repos/system_upgrade/el8toel9/actors/postgresqlcheck/libraries/postgresqlcheck.py

```diff
@@ -0,0 +1,55 @@
+from leapp import reporting
+from leapp.libraries.common.rpms import has_package
+from leapp.libraries.stdlib import api
+from leapp.models import InstalledRedHatSignedRPM
+
+
+# Summary for postgresql-server report
+report_server_inst_summary = (
+    'PostgreSQL server component will be upgraded. Since RHEL-9 includes only'
```

Comment:
A slight nitpick -- thinking about this long-term, there will be more versions at some point and then this text could be misleading. I'm thinking whether we can figure out some wording that will not need to be changed when we add more alternative versions. Maybe just remove the "only" word is good start.

Reply:
@hhorak good point. I've been thinking about that couple of time and I have to agree I do not have better way than what you suggested. (e.g. explaining that properly requires much more text and it could be confusing for user probably). The issue is already resolved in the proposed way, so resolving conversation :)